app/controllers/logs_controller.rb

```diff
@@ -271,19 +271,23 @@ def update
   # can be given a single id or a list of ids
   def take
     unless params[:ids].present?
```

Comment:
I'm too late as this PR is meged, but `unless x.present?` / `else` can be inverted to `if x.blank?` / `else`. Easier to follow in my mind :)